api/envoy/extensions/quic/v3/crypto_stream.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
Actually, one tiny nit. I think both API and extensions should have distinct directories for each category of extension. So, there should be `api/envoy/extensions/quic/crypto_stream/v3` etc. (same in `source/extensions`)

Reply:
done